### PR TITLE
Add skip parameter

### PR DESCRIPTION
**Pull request by chonton**

### CODE REVIEW

File: src/main/java/com/cosium/code/format/AbstractModulMavenGitCodeFormatMojo.java [hunk 1]
```diff
@@ -14,6 +15,10 @@
  */
 public abstract class AbstractModulMavenGitCodeFormatMojo extends AbstractMavenGitCodeFormatMojo {
 
+  /** Skip execution of this goal */
+  @Parameter(property = "fmt.skip", defaultValue = "false")
+  private boolean skip;
```
Comment (reda-alaoui) [inline src/main/java/com/cosium/code/format/AbstractModulMavenGitCodeFormatMojo.java]: Seems to me that `includedModules` and `excludedModules` achive the same thing?

**Comment by chonton** [comment]
What other information do you need?

**Comment by reda-alaoui** [comment]
> Seems to me that includedModules and excludedModules achieve the same goal?
An answer to the question above :)

**Comment by chonton** [comment]
```<excludedModules>*</excludedModules>``` could achieve the same behavior for all goals except `install-hook`

Consider the situation where I inherit a corporate pom that turns on this plugin's goals and I need to turn off one of the goals in my project.  This will require re-configuration of the plugin in my project (or setting the `excludedModules` property in ```<properties>```.)

When turning off a plugin, I look for a `skip` parameter which most plugins offer.

File: src/main/java/com/cosium/code/format/AbstractModulMavenGitCodeFormatMojo.java [hunk 2]
```diff
@@ -14,6 +15,10 @@
  */
 public abstract class AbstractModulMavenGitCodeFormatMojo extends AbstractMavenGitCodeFormatMojo {
 
+  /** Skip execution of this goal */
+  @Parameter(property = "fmt.skip", defaultValue = "false")
```
Comment (reda-alaoui) [inline src/main/java/com/cosium/code/format/AbstractModulMavenGitCodeFormatMojo.java]: Please use `gcf` instead of `fmt`

**Comment by reda-alaoui** [comment]
Please resolve the conflict so we can merge this 👍

**Comment by chonton** [comment]
Done

**Comment by reda-alaoui** [comment]
I am still seeing a merge conflict

**Comment by chonton** [comment]
really done

**Comment by reda-alaoui** [comment]
Thanks 👍

**Pull request merged**